### PR TITLE
HMIS MCH 001 Antenatal Card

### PR DESCRIPTION
**Pull request by Mwanje**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml [hunk 1]
```diff
@@ -0,0 +1,1493 @@
...
+
+    <span class="hidden">
+        <encounterLocation default="e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b" order="e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b" />
+        <encounterProviderAndRole encounterRole="240b26f9-dd88-4172-823d-4a8bfeb7841f" required="required" default="currentUser"/>
+    </span>
+    <!-- Beging of HTML Form -->
+    <div>
+        <div class="row">
+            <div class="col-md-6">
+                <h1>HMIS MCH 001 Antenatal Card</h1>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml]: Use HMIS MCH 001: Antenatal Card

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml [hunk 2]
```diff
@@ -0,0 +1,1493 @@
...
+                                        <div class="card-body">
+                                            <div class="row">
+                                                <div class="col-md-3">
+                                                    <obs conceptId="166196" labelText="WOA" />
+                                                </div>
+                                                <div class="col-md-3">
+                                                    <obs conceptId="166197" labelText="Fundal Height" />
+                                                </div>
+                                                <div class="col-md-3">
+                                                    <obs conceptId="166198" labelText="Presentation/Lie" />
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml]: Remove lie

**Comment by semujju** [comment]
@Mwanje 

- Health of Husband/Partner is missing under social history
- EMTCT Code is missing under progress examination
- Previous Obstetric History Section is missing
- Ultrasound Reports Section is missing
- Remove Diagnosis card

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml (hunk 1 above)
Comment (Mwanje) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml (hunk 2 above)
Comment (Mwanje) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-001-AntenatalCard.xml]: Done

**Comment by Mwanje** [comment]
@semujju
--Health of Husband/Partner concept is missing
--Previous Obstetric history  section is  under pregnancy information tab
--Ultra Sound Report is under counselling and investigation tab

**Comment by solemabrothers** [comment]
@Mwanje please compare the HardCopy tool with what is in the system, there are many fields on the card that shouldn't be there.

**Comment by solemabrothers** [comment]
@Mwanje for you consideration

1. Change Nutrition Assessment to Physical Examination and move the information you tagged as a physical examination into the section of Nutrition assessment.
2. Present Pregnancy information is missing.
3. Remove all investigations and only reflect those indicated on the card

File: api/src/main/java/org/openmrs/module/ugandaemr/api/deploy/bundle/CommonMetadataBundle.java [hunk 3]
```diff
@@ -69,6 +69,7 @@ public void install() throws Exception {
         install(EncounterTypes.COVID19_ENROLLMENT);
         install(EncounterTypes.COVID19_FOLLOWUP);
         install(EncounterTypes.COVID19_DISCHARGE);
+        install(EncounterTypes.ANC_CARD_ENCOUNTER_TYPE);
```
Comment (solemabrothers) [inline api/src/main/java/org/openmrs/module/ugandaemr/api/deploy/bundle/CommonMetadataBundle.java]: @Mwanje why create a new encounter when we already have these existing in UgandaEMR

File: api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java [hunk 4]
```diff
@@ -308,4 +308,20 @@ public String uuid() {
             return "afcdfcd8-defe-11eb-ba80-0242ac130004";
         }
     };
+
+    public static EncounterTypeDescriptor ANC_CARD_ENCOUNTER_TYPE = new EncounterTypeDescriptor() {
```
Comment (solemabrothers) [inline api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java]: @Mwanje You dont need this descriptor as the encounter is already existing

Comment (Mwanje) [reply api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java]: Done

File: api/src/main/java/org/openmrs/module/ugandaemr/api/deploy/bundle/CommonMetadataBundle.java (hunk 3 above)
Comment (Mwanje) [reply api/src/main/java/org/openmrs/module/ugandaemr/api/deploy/bundle/CommonMetadataBundle.java]: Done

**Comment by slubwama** [comment]
@solemabrothers  whats the hold on this one from being merged??

**Pull request closed without merging**